builder/builder.go

```diff
@@ -58,81 +58,115 @@ func getAuthData(filename string) (*dockerAuthData, error) {
 }
 
 // Build Docker Image.
-func (imgBuilder ImageBuilder) BuildImage() error {
+func (imgBuilder ImageBuilder) BuildImage(assignmentEnv bool) error {
```

Comment:
Also, it might be better to have two different functions.
1. `func (b ImageBuilder) BuildCodeRunnerImage() error`.
2. `func (b ImageBuilder) BuildAssignmentEnvImage() error`.

Reply:
Okay

Reply:
how is build context different?
For some reason passing a bool like that seems odd. As it's an indicator of sorts, the image builder should be aware of which image layer is actually to be built.

Reply:
Agreed. I have removed the bool variable